src/_styles/theme.ts

```diff
@@ -1,14 +1,23 @@
 export const theme = {
   // Use hashes
   colors: {
-    background: '#eaecec',
+    background: '#EAECEC',
     blurredBlack: 'rgba(0, 0, 0, 0.3)',
+    button: '#168b8d',
+    buttonActive: '#0f6163',
+    darkGreen: '#0B5641',
+    darkRed: '#A32941',
     grey: '#6e7777',
-    lightGreen: '#90D9DA',
+    lightBlue: '#90D9DA',
+    lightGreen: '#81D5BE',
+    lightGrey: '#CFD3D3',
+
     primary: '#39ac8d',
     primaryDark: '#127173',
+    red: '#F82E58',
     secondary: '#149773',
     secondaryDark: '#118363',
+    teal: '#1AA5A8',
```

Comment:
Might be nitpicking, but I find it more readable if you use lowercase letters in HEX values

Reply:
indeed nitpicking 😉